src/library_glemu.js

```diff
@@ -3674,6 +3673,7 @@ var LibraryGLEmulation = {
   },
 
   glLoadMatrixf: function(matrix) {
+    {{{ from64('matrix') }}}
```

Comment:
I think in other places we use the signature for this, and `j` would make it work automatically? We only seem to use explicit `from64()` calls in places we want to avoid changes to the call stack, from reading the code around `grep -U2 from64 src/*`.

Reply:
I think you mean `p` right?

I think there was a specific problem using `__sig` here which meant I had to fall back to this instead.   I forgotten what it was exactly now but I will go back and figure it out and add an explanation as to why `__sig` doesn't work here.

Reply:
(sorry, yes, I meant `p`)

Reply:
`__sig` seems to work now!  (I think one of my recent changes might have fixed that :)